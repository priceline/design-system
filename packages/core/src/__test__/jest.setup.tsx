import '@testing-library/jest-dom'
import 'jest-styled-components'

global.requestAnimationFrame =
  global.requestAnimationFrame ||
  function _raf(cb) {
    return setTimeout(cb, 0)
  }
