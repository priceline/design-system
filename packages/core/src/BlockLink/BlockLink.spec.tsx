import { render } from '../__test__/testing-library'
import { BlockLink } from './BlockLink'

describe('BlockLink', () => {
  test('renders', () => {
    const { asFragment } = render(<BlockLink>raw text</BlockLink>)
    expect(asFragment()).toMatchSnapshot()
  })
})
