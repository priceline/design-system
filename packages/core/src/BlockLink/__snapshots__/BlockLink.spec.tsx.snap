// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlockLink renders 1`] = `
.c0 {
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #0068ef;
}

.c0:hover {
  color: #049;
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.c1 {
  display: block;
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
}

<a
  className="c0 c1"
  color="primary"
  rel={null}
  size="medium"
>
  raw text
</a>
`;
